src/applications/vre/28-1900/wizard/pages/veteran/03-noActiveDutySeparation.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
This component and the one above have identical content. Is there a way to just use one for both wizard routes?

Reply:
The content in these may change so until its final I figured we should hold off on refactoring like this.